### DISCUSSION TITLE
Please help me troubleshoot why my IntelliJ IDEA keeps indexing the project continuously for a long duration.

### DISCUSSION THREAD

**Discussion by will-zdu** [open-telemetry/opentelemetry-java-instrumentation#8767] [Q&A] [answered]
I used JetBrains idea to open our project，How do you set up your projects to avoid long indexing times every time you open them?

**Comment by mateuszrzeszutek** [comment, marked as answer]
Weeell, you don't. This project is quite enormous and we haven't managed to circumvent the ridiculously long indexing times, we all suffer from that. We had some ideas about splitting the project and moving some of the less popular instrumentations over to the contrib repo, but we haven't tried that out yet.

**Comment by trask** [comment]
See #9757
